commons/error_constants.py

```diff
@@ -0,0 +1,104 @@
+                      "operation: Unknown")
+CORTX_ERR_WRONG_JSON = ("An error occurred (InvalidPart) when calling the CompleteMultipartUpload "
+                        "operation: One or more of the specified parts could not be found. The "
+                        "part might not have been uploaded, or the specified entity tag might not "
+                        "have matched the part's entity tag.")
+RGW_ERR_DUPLICATE_BKT_NAME = ("An error occurred (BucketAlreadyExists) when calling the "
+                              "CreateBucket operation: Unknown")
+CORTX_ERR_DUPLICATE_BKT_NAME = ("An error occurred (BucketAlreadyOwnedByYou) when calling"
+                                "the CreateBucket operation: Your previous request to create "
+                                "the named bucket succeeded and you already own it.")
+RGW_ERR_COPY_OBJ = ("An error occurred (InvalidRequest) when calling the CopyObject operation:"
+                    " The specified copy source is larger than the maximum allowable size for a "
+                    "copy source: 5368709120")
+CORTX_ERR_COPY_OBJ = ("An error occurred (InvalidRequest) when calling the CopyObject operation:"
+                      " The specified copy source is larger than the maximum allowable size for a "
+                      "copy source: 5368709120")
+RGW_ERR_COPY_OBJ_METADATA = ("An error occurred (InvalidRequest) when calling the CopyObject "
+                             "operation: This copy request is illegal because it is trying to "
+                             "copy an object to itself without changing the object's metadata, "
+                             "storage class, website redirect location or encryption attributes.")
+CORTX_ERR_COPY_OBJ_METADATA = ("An error occurred (InvalidRequest) when calling the CopyObject "
+                               "operation: This copy request is illegal because it is trying to "
+                               "copy an object to itself without changing the object's metadata, "
+                               "storage class, website redirect location or encryption attributes.")
+RGW_HEAD_OBJ_ERR = ("An error occurred (404) when calling the HeadObject operation: Not Found")
+CORTX_HEAD_OBJ_ERR = ("An error occurred (404) when calling the HeadObject operation: Not Found")
+
+ACCOUNT_ERR = "attempted to create an account that already exists"
+ACCESS_DENIED_ERR_KEY = "AccessDenied"
+CANNED_ACL_GRANT_ERR = "Specifying both Canned ACLs and Header Grants is not allowed"
+NOT_FOUND_ERR_CODE = "404"
+DUPLICATE_USER_ERR_KEY = "EntityAlreadyExists"
+INVALID_ACCESSKEY_ERR_KEY = "InvalidAccessKeyId"
+NO_BUCKET_OBJ_ERR_KEY = "NoSuchBucket"
+NO_SUCH_KEY_ERR = "NoSuchKey"
+NO_SUCH_UPLOAD_ERR = "NoSuchUpload"
+NO_BUCKET_NAME_ERR = "Required parameter bucket_name not set"
+NOT_FOUND_ERR = "Not Found"
+
+#S3 Bucket Tagging
+BKT_SET_TAG_ERR = "NoSuchTagSetError"
+BKT_INVALID_TAG_ERR = "InvalidTagError"
+BKT_TAG_DUPLICATE_KEY_ERR = "MalformedXML"
```

Comment:
Can we name it MALFORMED_XML ,so name will specify what is error string

Reply:
This error is related to bucket tag operation and hence named accordingly

Reply:
I think @archanabpatil  is write. Duplicate key error should have a different text.  We can rename this variable as MALFORMED_XML.